samples/firefox-accounts/src/main/java/org/mozilla/samples/fxa/MainActivity.kt

```diff
@@ -45,6 +45,14 @@ open class MainActivity : AppCompatActivity(), LoginFragment.OnLoginCompleteList
                 Config.custom(CONFIG_URL).then { value: Config ->
                     val acct = FirefoxAccount(value, CLIENT_ID, REDIRECT_URL)
                     account = acct
+
+                    val pairingUrl = getIntent().getExtras().getString("pairingUrl") ?: ""
```

Comment:
@csadilek would you like to use anything different here from `getIntent().getExtras()` 

Currently the flow is the following:

```java
MainActivity ------> ScanActivity ----- [set extras] ---------V
      ^                                                                                              
      |-------------------------------------------------------V
```

Reply:
Seems OK to me! I would probably introduce a constant for "pairingUrl" though.